XenAdmin/SettingsPanels/NRPEEditPage.cs

```diff
@@ -325,24 +322,18 @@ private string AllowHostsWithoutLocalAddress(string allowHosts)
                 UpdatedAllowHosts.Substring(0, UpdatedAllowHosts.Length - 1);
         }
 
-        private void UpdateOtherComponentBasedEnableNRPECheckBox(bool check)
+        private void UpdateOtherComponentBasedEnableNRPECheckBox()

```

Comment:
This is an instance method and the parameter passed in it is EnableNRPECheckBox.Checked which can be chekced directly within the method, so there is no need to pass it as a parameter; the code is thus simplified.